rebar.config

```diff
@@ -1,17 +1,38 @@
-{deps, []}.
+{deps, [
+    {erlang_lorawan, ".*",
+        {git, "https://github.com/helium/erlang-lorawan.git", {branch, "master"}}},
+    {grpc_client,
+        {git, "https://github.com/Bluehouse-Technology/grpc_client.git", {branch, "master"}}},
+    {grpcbox, {git, "https://github.com/novalabsxyz/grpcbox.git", {branch, "master"}}},
+    {helium_proto, {git, "https://github.com/helium/proto.git", {branch, "master"}}},
```

Comment:
I don't think it is a good idea to specify a dependency via branch name. This practice leaves us open to accidentally absorbing library changes that weren't intended for release.
Yes, I know that the change won't be absorbed unless _rebar.lock_ is updated, but that has two weaknesses:

1.  The risk of accidentally committing the change is increased when other dependency changes are in _rebar.lock_.
2. Nothing is communicated about the  _intent_ of the change. When a tag (or at minimum, a commit hash) is used, it is clear that the change was consciously made.

We don't live in a world where mistakes are never committed to significant branches, nor where our testing schemes are likely to save us from not thinking about these changes.